apps/test/unit/templates/instructions/ReviewTabTest.jsx

```diff
@@ -12,62 +12,71 @@ import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import Button from '@cdo/apps/templates/Button';
 
 describe('Code Review Tab', () => {
-  const token = 'token';
   const channelId = 'test123';
   const serverLevelId = 123;
   const serverScriptId = 123;
   const reviewableProjectId = 'reviewableProjectId123';
-  const existingComment = Factory.build('CodeReviewComment');
-  let wrapper, server, clock, onLoadComplete;
+
+  const autoResolveApiCall = () => {
```

Comment:
Yeah definitely agreed - stubbing a promise-like function call is really verbose. From my brief search, I couldn't find any recommended solutions for mocking promise-like functions but not actually using promises in tests, but if there is something it would be nice to clean this up a bit.